server/views/components/image/index.njk

```diff
@@ -0,0 +1,38 @@
+<figure class="figure
+               {{ ' figure--full' if full }}
```

Comment:
I think this will output on multiple lines. Is that OK for our supported browsers (I've no idea).